frontend/src/css/App.css

```diff
@@ -14,29 +14,56 @@ All component specific CSS lives in the React file with the component.
 /* Headers */
 h3 {
   color: #009fae;
```

Comment:
@joeyhua we're using this colour for text a lot (also in the progress bubbles), but it's not an accessible contrast.

<img width="729" alt="screen shot 2019-02-27 at 7 47 14 am" src="https://user-images.githubusercontent.com/4640747/53491286-0e25a700-3a64-11e9-8c7e-905bb9b1fbc7.png">

Reply:
Let's look at colours in a separate PR since you didn't actually change this - my bad!